src/main/java/com/google/swarm/tokenization/common/DLPTransform.java

```diff
@@ -143,9 +148,9 @@ public PCollectionTuple expand(PCollection<KV<String, Table.Row>> input) {
                       .build())
               .apply(
                   "ConvertDeidResponse",
-                  ParDo.of(new ConvertDeidResponse())
+                  ParDo.of(new ConvertDeidResponse(dataSinkType()))
                       .withOutputTags(
-                          Util.inspectOrDeidSuccess, TupleTagList.of(Util.inspectOrDeidFailure)));
+                          Util.inspectOrDeidSuccess, TupleTagList.of(Util.inspectOrDeidFailure).and(Util.deidSuccess)));
```

Comment:
Why Util.deidSuccess tag? For both Inspect and Deid same output tags are being used so why additional deidSuccess tag is needed?